.github/workflows/ci-conda.yml

```diff
@@ -87,15 +87,12 @@ jobs:
           echo "::remove-matcher owner=configure-system-package-warning::"
           echo "::remove-matcher owner=configure-system-package-error::"
 
-          # Manually install ptyprocess for now, until https://github.com/sagemath/sage/issues/32147 is fixed
-          pip install --no-build-isolation -v -v ptyprocess==0.5.1
```

Comment:
Some other conda package is bringing in pytprocess

Reply:
We pip-install `ptyprocess` via the src`/environment.yml`

Reply:
The dependency check fails though.

Reply:
Indeed; I'll check what's happening

Reply:
Fixed in e919263; but now it emits a few more `pip` lines for other packages too.